### ISSUE TITLE
Новый термин - well-known URI

### ISSUE THREAD

**Issue by skip405** [web-standards-ru/dictionary#454]
https://en.wikipedia.org/wiki/Well-known_URI

Описание, которое помогает понять суть:

> They are URL path prefixes that start with `/.well-known/`. This implementation is in response to the common expectation for web-based protocols to require certain services or information be available at URLs consistent across servers, regardless of the way URL paths are organized on a particular host.

Термин пока не сильно распространён в русскоязычном сегменте, [в статье на Хабре](https://habr.com/ru/company/timeweb/blog/525864/) используют `well-known URL`.

Мне видится подходящим использование прилагательных "общепринятый", "сложившийся", "стандартный" - что-то тип того. Расположил, как мне кажется, в порядке от хорошего варианта к плохому.

Нужно ли добавлять в словарь? Есть у кого-нить мысли по термину и о том, как его можно переводить?

**Comment by pepelsbey** [comment]
Мне кажется, что этот скорее как .htaccess — вряд ли его стоит вот прямо переводить. Это просто название папки, другое дело, что можно подыскать и перевести название этого подхода, может оно как-то иначе называется.

**Comment by pepelsbey** [comment]
Ну и как водится я бы начинал с примеров использования и пытался бы перевести в контексте. Так сразу становится понятно, какие особенности есть у слова и какие есть шансы на его перевод.

**Comment by skip405** [comment]
Да, совсем забыл.

- A list of well-known URLs can be found on iana.org.
- Well-known URLs provide well-known resources.
- To register a new well-known URL you should provide a name relative to the `/.well-known/` part.

Да, судя по всему, переводить такое не нужно, но как писать, может в кавычках?)) — «Давайте добавим „well-known“ ссылку на наш сайт».

Если принять во внимание, что URL тоже может писаться без перевода, то подобные фразы вообще могут выглядеть диковато, по крайней мере для меня — «Давайте добавим well-known URL на сайт». Не хватает только мемчика с той женщиной из Америки и интонацию менять при чтении.

**Comment by pepelsbey** [comment]
К сожалению, уже при втором употреблении перевод «общепризнанный» потеряет специфический смысл и будет восприниматься в обычном смысле. Боюсь, что шансов на перевод пока нет.

Я бы писал:

- Список well-known ссылок можно найти на iana.org
- Для регистрации well-known ссылки нужно добавить…

**Comment by skip405** [comment]
Да, но в теории это проблема этого прилагательного. Например, заменим его на «консистентный». И сама специфика прилагательного будет менять смысл. Не? Да, я помню, что мы не придумываем переводы, Вадим ))

— Чтобы зарегистрировать консистентную ссылку нужно…
— Какую-какую ссылку?
— Консистентную, ну, ту, которая well-known
